understanding/headings-and-labels.html

```diff
@@ -58,190 +56,133 @@
-                  may suffice if it provides an appropriate cue to finding and navigating content.
+               <p>ラベル及び見出しを長くする必要があるわけではない。単語一つ又は一文字だけであっても、コンテンツを見つけてナビゲートする手がかりとして適切であれば、それだけで十分なこともある。
                   
                </p>
-               <p>This Success Criterion does not require headings or labels. This Success Criterion
-                  requires that if headings or labels are provided, they be descriptive. This Success
-                  Criterion also
-                  does not require that content acting as a heading or label be correctly marked up
-                  or
-                  identified – this aspect is covered separately by
-                  <a href="info-and-relationships">1.3.1: Info and Relationships</a>. It is possible for content
-                  to pass this Success Criterion (providing descriptive content that acts as headings
-                  or labels) while failing
-                  Success Criterion 1.3.1 (if the headings or labels aren't correctly marked up/identified).
-                  Conversely,
-                  it is also possible for content to pass Success Criterion 1.3.1 (with headings or
-                  labels correctly
-                  marked up or identified), while failing this Success Criterion (if those headings
-                  or labels are not
-                  sufficiently clear or descriptive).
-                  
+               <p>この達成基準は見出しやラベルを必要としない。見出しやラベルが提供されている場合、それらの内容が分かるように記述されていることが求められる。この達成基準は見出し又はラベルとして機能するコンテンツを正しくマークアップすること又は識別することも必要としない。この点は <a href="info-and-relationships">1.3.1: 情報及び関係性</a>で別途取り扱っている。コンテンツがこの達成基準に合格しながら (見出し又はラベルとして機能する説明的なコンテンツを提供している)、達成基準 1.3.1 (見出し又はラベルを誤ってマークアップ / 識別している場合) に失敗することがありえる。反対に、コンテンツが達成基準 1.3.1 に合格しながら (見出し又はラベルが正しくマークアップ又は識別されている)、この達成基準に失敗することも可能である (それらの見出し又はラベルが十分に明確又は説明的ではない場合)。
+
                </p>
-               <p>Further, in the case of labels, this Success Criterion does not take into consideration
-                  whether or not
-                  alternative methods of providing an accessible name for form controls and inputs has
-                  been
-                  used – this aspect is covered separately by <a href="name-role-value">4.1.2: Name, Role and Value</a>. It is possible
-                  for controls and inputs to have an appropriate accessible name (e.g. using <code>aria-label="…"</code>)
-                  and therefore pass Success Criterion 4.1.2, but to still fail this Success Criterion
-                  (if the label is not
-                  sufficiently clear or descriptive).
+               <p>さらに、ラベルの場合、この達成基準はフォームコントロール及び入力にアクセシブルな名前を提供する代替手段が用いられているか否かを考慮しない。この点は <a href="name-role-value">4.1.2: 名前 (name)・役割 (role)・値 (value)</a> で別途取り扱っている。コントロール及び入力に適したアクセシブルな名前 (例えば、<code>aria-label="…"</code> を使用して) を持ち、その結果達成基準 4.1.2 に合格したとしても、(ラベルが十分に明確又は説明的ではない場合) この達成基準に失敗する可能性がある。
```

Comment:
https://github.com/waic/wcag22/pull/392#discussion_r2083826842 で似たような文があるので書きぶりを揃えてしまいますか。

```suggestion
               <p>さらに、ラベルの場合、この達成基準はフォームコントロール及び入力にアクセシブルな名前を提供する代替手段が用いられているか否かを考慮しない。この点は <a href="name-role-value">4.1.2: 名前 (name)・役割 (role)・値 (value)</a> で別途取り扱っている。コントロール及び入力欄が適切なアクセシブルな名前 (accessible name) を (例えば <code>aria-label="..."</code> を使用して) 持っており、これによって達成基準 4.1.2 を満たすが、この達成基準には失敗するということもありえる (ラベルが十分に明確又は説明的ではない場合)。
```